sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_models.py

```diff
@@ -233,3 +336,16 @@ def __init__(
         self.file_extensions = kwargs.get('file_extensions', None)
         self.content_types = kwargs.get('content_types', None)
         self.versions = kwargs.get('versions', None)
+
+    @classmethod
+    def _from_generated(cls, file_format):
+        return cls(
+            format=file_format.format,
+            file_extensions=file_format.file_extensions,
+            content_types=file_format.content_types,
+            versions=file_format.versions
+        )
+
+    @classmethod
+    def _from_generated_list(cls, file_formats):
+        return list( [ FileFormat._from_generated(file_formats) for file_formats in file_formats ] ) 
```

Comment:
Believe the generated code returns a model which we need to .value into to get the iterable

```suggestion
        return [FileFormat._from_generated(file_formats) for file_formats in file_formats.value ]
```